code/game/objects/items/weapons/grenades/ghettobomb.dm

```diff
@@ -110,6 +174,17 @@
     assembled = 2
     active = 0
 
+/obj/item/weapon/grenade/iedcasing/preassembled/withshrapnel
+	name = "shrapnel loaded improvised explosive"
+
+/obj/item/weapon/grenade/iedcasing/preassembled/withshrapnel/New()
+	..()
+	shrapnel_list.Add(new /obj/item/weapon/shard(src))
```

Comment:
I still recommend a loop
But now I also recommend you write a nice, simple proc to add shrapnel to the IED
All it has to do is move the item into the IED, add it to the shrapnel list, and add its size to `current_shrapnel`
That way you don't have to hardcode the value of `current_shrapnel` here, and you can make it possible for shrapnel to be added by other means without copypaste.